### DISCUSSION TITLE
Problème vstream et source uptobox

### DISCUSSION THREAD

**Discussion by domi74000** [Kodi-vStream/venom-xbmc-addons#3994] [Support & aide] [answered]
Bonjour, j'ai un compte uptobox premium. J'ai entré mon compte dans les paramètres de vstream. Jusqu'à avant hier, en allant sur vstream-sources-compte uptobox, j'avais accès à tous mes fichiers d'uptobox. Depuis hier, ça ne fonctionne plus, et pourtant mon compte uptobox est toujours valide.
Est ce quelqu'un peut m'aider svp.
Merci

**Comment by sizanic** [comment]
Bonjour,

depuis hier, uptobox à mis à jour une nouvelle version avec des évolutions.

Ce qui ne permet plus à vStream de proposer les fichiers présents sur le compte.

Mais la correction est en cours, une nouvelle version de vStream pour prendre en compte cette évolution va sortir très rapidement.

**Comment by sizanic** [comment, marked as answer]
Comme il s'agit d'un probleme avéré, on ne va pas continuer la discussion ici, mais au travers de ce ticket : 
https://github.com/Kodi-vStream/venom-xbmc-addons/issues/3995
